pkg/common/store/nodeslices.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2024 The Godel Scheduler Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package store
+
+import (
+	framework "github.com/kubewharf/godel-scheduler/pkg/framework/api"
+)
+
+var GlobalNodeInfoPlaceHolder = framework.NewNodeInfo()
+
+type NodeSlices struct {
```

Comment:
Please move these code to pkg/framework/api/nodeinfo_hashslice.go

The data structure is not bound with the CommonStore Mechanism.

Reply:
Ok, I have move it to `pkg/framework/api/nodeinfo_hashslice.go`.